### PR TITLE
add pa exception messages by organization

### PR DESCRIPTION
**Pull request by ritabulman**
the messages come from this google sheet: https://docs.google.com/spreadsheets/d/1X6b1Pyio8FC2ZpLaKBfmlkEjsWKuYov-YhFe2k3HG6Q/edit?gid=0#gid=0

### CODE REVIEW

**Review by samkotlove — COMMENTED** [review]
this doesnt look like it's getting used anywhere?

**Comment by jknapp25** [comment]
do you mind adding to types with description?

File: packages/settings/src/lib/photon.ts [hunk 1]
```diff
@@ -148,7 +156,9 @@ const organizationSettings: {
   org_v5It8IoY0RH1Rw80: {
     logo: 'piction_health_logo.png',
     accentColor: '#3377e2',
-    enableRxAndOrder: false
+    enableRxAndOrder: false,
+    paExceptionMessage:
+      'Your insurance needs additional information from your provider before you pharmacy can fill your prescription. Please contact 781-650-4492 and ask them to submit a prior authorization on your behalf or suggest an alternative medication.'
```
[the remark this answers is not in the document]
Comment (ritabulman) [reply packages/settings/src/lib/photon.ts:161]: i have looked at your comment for a long time and i am confused about what you mean lol, are you saying i should add an extra space here: `781-650-4492 <>and`?

[the remark this answers is not in the document]
Comment (jknapp25) [reply packages/settings/src/lib/photon.ts:161]: oh sorry!, it just looked like a doublespace was there, just a nit

**Review by jknapp25 — APPROVED** [review]

**Pull request merged**